### PR TITLE
Jipm/header update juniper

### PR DESCRIPTION
**Pull request by jignaciopm**
update from ironwood to juniper all header templates

### CODE REVIEW

**Comment by morenol** [comment]
Looks good @jignaciopm. Could you add screenshots of that to the PR description? You can take them from the devstack. A screenshot before the change and a screenshot after the change

File: edx-platform/msus-produhktiv/lms/templates/header/header.html [hunk 1]
```diff
@@ -92,4 +114,4 @@
             </label>
         </form>
     % endif
-% endif
+% endif
```
Comment (AuraAlba) [inline edx-platform/msus-produhktiv/lms/templates/header/header.html]: @jignaciopm there's missing a new line at the end of the file

File: edx-platform/produhktiv/lms/templates/header/header.html [hunk 2]
```diff
@@ -92,4 +114,4 @@
             </label>
         </form>
     % endif
-% endif
+% endif
```
Comment (AuraAlba) [inline edx-platform/produhktiv/lms/templates/header/header.html]: @jignaciopm new line too

**Review by AuraAlba — CHANGES_REQUESTED** [review]

File: edx-platform/msus-produhktiv/lms/templates/header/header.html (hunk 1 above)
Comment (jignaciopm) [reply edx-platform/msus-produhktiv/lms/templates/header/header.html]: @amalbas ready on my last commit

File: edx-platform/produhktiv/lms/templates/header/header.html (hunk 2 above)
Comment (jignaciopm) [reply edx-platform/produhktiv/lms/templates/header/header.html]: @amalbas ready on my last commit

**Review by AuraAlba — APPROVED** [review]

**Comment by AuraAlba** [comment]
@morenol there is no change in the look and feel of the header. There are changes for attributes or sections that are not common to use. So I don't think the screenshots will have differences.

**Review by morenol — APPROVED** [review]

**Pull request merged**